opendcs-rest-api-jetty/Dockerfile-install-postgres-schema

```diff
@@ -6,6 +6,7 @@ RUN apt-get install openjdk-17-jre -y
 COPY ./docker-config/ /opendcs/
 COPY ./build/tmp/opendcs-installer.jar /opendcs/opendcs-installer.jar
 COPY ./docker-config/InstallTsdbSchemaDocker.sh /docker-entrypoint-initdb.d/InstallTsdbSchemaDocker.sh
+RUN chmod 777 /docker-entrypoint-initdb.d/InstallTsdbSchemaDocker.sh
```

Comment:
Do permissions need to be set here?

Reply:
yeah that was part of the issue running locally. Didn't help when running through GitHub Actions though

Reply:
Got it....We can leave it.